Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/AdaptiveHoughTransformSeeder.hpp

```diff
@@ -0,0 +1,478 @@
+    /// the detector to use different algorithms for space point construction,
+    /// e.g. single-hit space points for pixel-like detectors or double-hit
+    /// space points for strip-like detectors.
+    /// Note that we don't *need* spacepoints (measurements can be used instead)
+    std::vector<std::string> inputSpacePoints;
+    /// Output track seed collection.
+    std::string outputSeeds;
+    /// Output hough track collection.
+    std::string outputProtoTracks;
+    /// Tracking geometry required to access global-to-local transforms.
+    std::shared_ptr<const Acts::TrackingGeometry> trackingGeometry;
+    float phiWrap =
+        0.1f;  // wrap size around angle domain limits (min pT dependent)
+    float qOverPtMin = 0.9f;  // min q/pt, -1/1 GeV
+
+    float qOverPtMinBinSize = 0.01f;  // minimal size of pT bin that the
+                                      // algorithm should not go beyond (in GeV)
+    float phiMinBinSize = 0.01f;  // minimal size of phi bin that the algorithm
+                                  // should not go beyond
+    float zRange = 200;           // range in z
+    float cotThetaRange = 10;     // range in cotTheta
+
+    float zMinBinSize =
+        1;  // minimal size of z bin that the algorithm should
+            // not go beyond when exploring zvertex-cot(theta) space space
+    float cotThetaMinBinSize = 0.1f;  // minimal size of cot(theta) bin that
+                                      // the algorithm should not go beyond
+    unsigned threshold =
+        4;  // number of lines passing section for it to be still considered
+    unsigned noiseThreshold = 12;  // number of lines passing section at the
+                                   // final split to consider it noise
+
+    bool doSecondPhase = true;  // do the second pass in z-cot(theta) space to
+                                // find less solutions
+    bool deduplicate = true;    // when adding solutions try avoiding duplicates
+
+    double inverseA =
+        1.0 / 3.0e-4;  // Assume B = 2T constant. Can apply corrections to
+                       // this with fieldCorrection function
+                       // This 3e-4 comes from the 2T field when converted to
+                       // units of GeV / (c*mm*e)
+  };
+
+  // information that is needed for each measurement
+  struct PreprocessedMeasurement {
+    /// Construct the measurement
+    /// @param inverseR inverse of radius of the SP
+    /// @param phiAngle azimuthal angle of the SP
+    /// @param zpos z position of the SP
+    /// @param l link to space point
+    PreprocessedMeasurement(double inverseR, double phiAngle, double zpos,
+                            Acts::SourceLink l)
+        : invr(inverseR), phi(phiAngle), z(zpos), link(std::move(l)) {}
+    double invr;
+    double phi;
+    double z;
+    Acts::SourceLink link;
+  };
+
+  template <typename M = PreprocessedMeasurement>
+  struct AHTExplorationOptions {
```

Comment:
Convention would be 
```suggestion
  struct AhtExplorationOptions {
```

although that's not super clear. Could also just be `ExplorationOptions` maybe?